docs/qualification-tool.md

```diff
@@ -83,7 +83,7 @@ any machine and include the jars in the classpath.
 java -cp ~/rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*:$HADOOP_CONF_DIR/ \
  com.nvidia.spark.rapids.tool.qualification.QualificationMain --help
 
-RAPIDS Accelerator for Apache Spark Qualification tool
+Qualification tool - RAPIDS Accelerator for Apache Spark
```

Comment:
lets change to RAPIDS Accelerator Qualification tool for Apache Spark

Reply:
done!